src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -47,6 +47,7 @@
 
   // Menu Item
   --pf-c-options-menu__menu-item--Background: transparent;
+  --pf-c-options-menu__menu-item--FontSize: var(--pf-global--FontSize--md);
```

Comment:
Not sure if you need to define 16px font-size here as it is that by default.

Reply:
I talked to @mceledonia about it and we don't want the menu item to inherit a `font-size` from a parent. For example, the options menu in the pagination - the pagination component is 14px, but the options menu items should still be 16px. This is also true with the dropdown and select, which have `font-size` set on their menu-items, too.

Reply:
oh cool i was just wondering if it would get overwritten. lets stick with it.